build/toolchain/android/BUILD.gn

```diff
@@ -65,7 +65,9 @@ template("android_gcc_toolchain") {
       cxx = compiler_prefix + tool_prefix + "g++"
     }
 
-    ar = tool_prefix + "ar"
+    # The NDK shipped ar utility is out of date, and using it will cause linker
+    # failures for version inconsistencies.
+    ar = "ar"
```

Comment:
Which version of `ar` will this map to?  Is it going to use the `ar` in the environment's `PATH`?

I tried doing an `android_release` build with Clang 11 using:
`rebase_path("//buildtools/$host_dir/clang/bin", root_build_dir) + "/llvm-ar"`

So far that seems to be working.

Reply:
This is how it's set up in the iOS toolchain - although yes, I think you're right.  I'll switch this to your suggestion so we're sure to get the right ar from buildtools.